package.json

```diff
@@ -57,11 +57,14 @@
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-1": "6.24.1",
     "css-loader": "0.28.11",
+    "devtron": "1.4.0",
     "electron": "2.0.0",
+    "electron-debug": "2.0.0",
```

Comment:
It should be a dependency, not devDependency.
Open the packaged app to see the error.

Reply:
> It should be a dependency, not devDependency.
> Open the packaged app to see the error.

:man_facepalming: fixed, thanks for catching that!